test/remote-connector.test.js

```diff
@@ -1,6 +1,7 @@
 var loopback = require('loopback');
 var defineModelTestsWithDataSource = require('./util/model-tests');
 var assert = require('assert');
+var Remote = require('../');
```

Comment:
Ah, I forgot `require('..')` works - I think `require('.')` doesn't... I'll fix this.